robolectric/src/test/java/org/robolectric/shadows/ShadowLocaleDataTest.java

```diff
@@ -12,6 +12,11 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 @RunWith(TestRunners.MultiApiWithDefaults.class)
+@Config(sdk = {
```

Comment:
It's not passing on 21, some broke because of resource management, some because of different class implementation of layout classes.